plugins/python-build/bin/python-build

```diff
@@ -1537,6 +1537,9 @@ build_package_mac_openssl() {
   local pem_file="$OPENSSLDIR/cert.pem"
   security find-certificate -a -p /Library/Keychains/System.keychain > "$pem_file"
   security find-certificate -a -p /System/Library/Keychains/SystemRootCertificates.keychain >> "$pem_file"
+
+  # Make sure we build Python with *this* OpenSSL
+  package_option python configure --with-openssl="$OPENSSL_PREFIX_PATH"
```

Comment:
It seems "--with-openssl" has been added since 3.7. I suspect this could make build of older versions failimg 🤔 

https://bugs.python.org/issue21541

Reply:
It is true that that option only exists since 3.7 (according to the documentation, anyway), however this change seems to work just fine with 3.6.7 and 2.7.15, both of which I tried.